src/instructions/ast.rs

```diff
@@ -1,445 +0,0 @@
-use crate::Register;
```

Comment:
I do recommend we keep this file around. It could be useful in numerous tools surrounding ckb-vm, while also providing an example that the register value of a VM instance could be customized.

Reply:
Done